Mathlib/Data/Fintype/Card.lean

```diff
@@ -761,12 +761,18 @@ noncomputable def Finset.equivFinOfCardEq {s : Finset α} {n : ℕ} (h : s.card
   Fintype.equivFinOfCardEq ((Fintype.card_coe _).trans h)
 #align finset.equiv_fin_of_card_eq Finset.equivFinOfCardEq
 
+theorem Finset.card_eq_of_equiv_fin {s : Finset α} {n : ℕ} (i : s ≃ Fin n) : s.card = n :=
+  Fin.equiv_iff_eq.1 ⟨s.equivFin.symm.trans i⟩
```

Comment:
Could you also add the generalization: if `β` is a `Fintype` and `s ≃ β`, then `s.card = Fintype.card β`? That might also be a slight shortcut towards your new `Finset.card_eq_of_equiv`.